### ISSUE TITLE
Left Menu Scroll is Overlapped by .layout-content div this makes the screen not scrollable.

### ISSUE THREAD

**Issue by manu244** [primefaces/primeflex#214]
Left Menu Scroll is Overlapped by .layout-content div this makes the screen not scrollable.

**Comment by manu244** [comment]
TO fix this  .layout-content margin-left should be 300px

**Comment by Da13Harris** [comment]
Yeah, to be clear, this is an issue with the primeflex.org website that makes it difficult to scroll the left menu on a desktop device. It's still scrollable with a mouse wheel, but clicking and dragging the scrollbar doesn't work due to the overlapping content.
